docs/features/techdocs/getting-started.md

```diff
@@ -230,7 +230,7 @@ You can do so by including the following lines in the last step of your
 
 ```Dockerfile
 RUN apt-get update && apt-get install -y python3 python3-pip
-RUN pip3 install mkdocs-techdocs-core==0.0.16
+RUN pip3 install mkdocs-techdocs-core==0.2.1
```

Comment:
I believe we also just released `0.2.2` a bit ago.  While you're updating things for DCO, could you also update the pinned version?

Reply:
done